frontends/web/src/app.module.css

```diff
@@ -0,0 +1,8 @@
+.parentContainer {
+    min-width: 0;
+    overflow: auto;
+}
+
+.banners > div:first-child {
+    margin-top: var(--space-half);
```

Comment:
I'm testing 8cda810 but can't see margin-top with space-half for the sdcard check, also there is div class="undefined" but probably not related.

<img width="1484" alt="Screenshot 2024-07-29 at 15 59 02" src="https://github.com/user-attachments/assets/28daa296-e70a-4df7-be40-400ecca83959">